src/gas-tap-lib.js

```diff
@@ -55,6 +55,9 @@ var GasTap = (function () {
       
       , throws: throws
       , notThrow: notThrow
+
+      , isNaN: isNaN
```

Comment:
According to the naming style like equal vs notEqual, ok vs notOk, I'd like to suggest to change the isNan to nan and isNotNan to notNan, how do you think about it?